matrix/docker/query_runner.py

```diff
@@ -1,9 +1,81 @@
 """Script to pull from sqs and run redshift queries. Will be dockerized."""
```

Comment:
Update comment?

Reply:
Hmm update to what? This is exactly what the query runner is doing

Reply:
Read "Will be dockerized" as a TODO, nevermind